cli/command/service/opts.go

```diff
@@ -430,6 +432,21 @@ func newServiceOptions() *serviceOptions {
 func (opts *serviceOptions) ToService() (swarm.ServiceSpec, error) {
 	var service swarm.ServiceSpec
 
+	envVariables, err := runconfigopts.ReadKVStrings(opts.envFile.GetAll(), opts.env.GetAll())
+	if err != nil {
+		return service, err
+	}
+
+	env := []string{}
+	// Remove any duplicates of entries ("K=V K=V"). Note that duplicates of Keys ("K=V1 K=V2") are kept intact.
```

Comment:
Hm, having the same key twice, would result in the first value being overwritten by the second one, so wondering if we should keep both, or simply de-duplicate by key